sanity-io/sanity
# fix(sanity, test): fix issue with disable action button, update test

### Description

Issue where opening a array editing view, the main action wasn't reactivated
This is related to the focus fix that was done, I have a suggestion but I'm unsure it's the right way to go @hermanwikner 

[video](https://sharing.clickup.com/clip/p/t9014411731/bca3cd21-1370-439c-9882-58432e28cdec/screen-recording-2024-07-23-19%3A11.webm)

### What to review

Solution 

### Testing

3. Open a document which contain an array of object
5. Open one of the object and edit a field inside this object
7. Close the popup
9. Notice that the button is disabled, even though we updated a field

11. ### Notes for release

Fixes issue where opening the beta array editing would keep the publish button disabled
